hs_core/models.py

```diff
@@ -2930,26 +3096,37 @@ def irods_url(self):
 
 
 class Bags(models.Model):
+    """Represent data bags format as django model."""
+
     object_id = models.PositiveIntegerField()
     content_type = models.ForeignKey(ContentType)
 
     content_object = GenericForeignKey('content_type', 'object_id', for_concrete_model=False)
     timestamp = models.DateTimeField(default=now, db_index=True)
 
     class Meta:
+        """Define meta properties of Bags model."""
+
         ordering = ['-timestamp']
 
     def get_content_model(self):
+        """Return content model of Bags' content object."""
         return self.content_object.get_content_model()
 
 
 class PublicResourceManager(models.Manager):
+    """Extend Django model Manager to allow for public resource access."""
+
     def get_queryset(self):
+        """Extend Django model Manager to allow for public resource access."""
         return super(PublicResourceManager, self).get_queryset().filter(raccess__public=True)
 
 
 class DiscoverableResourceManager(models.Manager):
+    """Extend Django model Manager to filter for public and discoverable resources."""
```

Comment:
... public and discoverable resources. -> ...public **or** discoverable resources.